RELEASE-NOTES.md

```diff
@@ -1,5 +1,7 @@
 ## RELEASE NOTES
-### Version 2.79.2
+### Version 2.79.3
```

Comment:
The version here doesn't match the one in `package.json`.

Reply:
the version is matched now